x-pack/platform/plugins/shared/task_manager/server/saved_objects/schemas/task.ts

```diff
@@ -64,6 +64,16 @@ export const taskSchemaV4 = taskSchemaV3.extends({
       apiKeyCreatedByUser: schema.boolean(),
     })
   ),
+  schedule: schema.maybe(
```

Comment:
can remove this, it's now in v5 below

Reply:
Missed it! Thanks for catching. Removed.